### DISCUSSION TITLE
Increase support for Spine software

### DISCUSSION THREAD

**Discussion by 1481351607** [godotengine/godot-proposals#8802] [2D]
Increase support for Spine software

**Comment by AThousandShips** [comment]
What kind of spine software? Support in what way? What's missing? Please provide any details 🙂

**Reply by Calinou** [reply]
OP is referring to [Spine](https://esotericsoftware.com/), which is proprietary 2D animation software.

**Comment by Calinou** [comment]
Due to Spine's proprietary nature and lack of a runtime under an open source license, we can't officially support it in core Godot. However, Spine officially develops a [Godot runtime](https://esotericsoftware.com/spine-godot) as a C++ module with precompiled builds provided.
